src/record-matching/bib/generate-query-list.js

```diff
@@ -33,11 +33,14 @@ export function generateIdentifierQueries(record) {
 	function getStandardIdentifiers() {
 		return record.get(/^020|022|024$/)
 			.reduce((acc, field) => {
```

Comment:
Prefer Array method calls instead of for-loops. You can filter out the non-relevant subfields and then reduce the array:
```js
return field.subfields
.filter(({code}) => code in codes)
.reduce((acc, {value}) => {
 // Check that value defined and unique
}, []);
```